src/modules/QuestsList/index.tsx

```diff
@@ -1,112 +1,146 @@
-import { useEffect, useState } from 'react'
-import { Link } from 'react-router-dom'
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
 import "./QuestsList.scss";
-import { QuestInProgress, CurrentQuests, ComponentPath, CardTypeObj, QuestEncounterFunctoinality } from '../../interfaces'
-import questCardActive from '../../assets/Quest Cards/QuestCard_Active.png'
-import questCardActiveH from '../../assets/Quest Cards/QuestCard_Active_Hover.png'
-import questCardPassive from '../../assets/Quest Cards/QuestCard_Passive.png'
-import questCardPassiveH from '../../assets/Quest Cards/QuestCard_Passive_Hover.png'
-import questCardSupportive from '../../assets/Quest Cards/QuestCard_Supportive.png'
-import questCardSupportiveH from '../../assets/Quest Cards/QuestCard_Supportive_Hover.png'
+import {
+  QuestInProgress,
+  CurrentQuests,
+  ComponentPath,
+  CardTypeObj,
+  QuestEncounterFunctoinality,
```

Comment:
Comma needed here? I don't know Typescript, clearly

Reply:
It's not technically necessary, but my linter puts them in automatically XD